pom.xml

```diff
@@ -217,6 +217,22 @@
         </dependency>
     </dependencies>
     <build>
+		<testResources>
+        	<testResource>
```

Comment:
@bdragan these 2 `testResource` groups look similar to each other. Could you please explain the logic for them?